docs/pvacbind/output_files.rst

```diff
@@ -20,16 +20,16 @@ created):
 
    * - File Name
      - Description
-   * - ``<sample_name>.all_epitopes.tsv``
+   * - ``<sample_name>.<MHC_Class/Combined>.all_epitopes.tsv``
```

Comment:
```suggestion
   * - ``<sample_name>.<MHC_I|MHC_II|Combined>.all_epitopes.tsv``
```
I think this wording is a bit clearer. If you agree, please update throughout the docs.